generate-language-platform-bundles/testing/download_bundles_and_test_run_all.sh

```diff
@@ -43,6 +43,7 @@ TARGET_LANGUAGES=${TARGET_LANGUAGES:-${AVAILABLE_LANGUAGES}}
 TARGET_TEST_RESULTS_FOLDER=""
 
 SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+SCRIPT_PARENT_DIR="$(realpath ${SCRIPT_CURRENT_DIR}/..)"
```

Comment:
Works fine on git-bash and Linux bash, is this okay for MacOS as well - I remember you mentioned the last time, that `realpath` isn't an issue.